src/gui/src/layoutViewer.h

```diff
@@ -126,6 +127,7 @@ class LayoutViewer : public QWidget
                const std::vector<std::unique_ptr<Ruler>>& rulers,
                std::function<Selected(const std::any&)> makeSelected,
                std::function<bool(void)> usingDBU,
+               std::function<bool(void)> showFps,
```

Comment:
Maybe this can be accessed via the tcl interface instead?

Reply:
I think we're trying to move away from using the TCL interface inside the application itself. Maybe we could package these into an object instead?

Reply:
hmm, maybe the solution would be to have an access to through the GUI to "trigger" these from swig so it would extend easier for python and be extensible as we add more layer. I'll make a note for future fixes.  Either way this is fine. If we choose to make it more of an debugging overlay, then the name would change but thats about it.